src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/BarnehagelisteVarslingService.kt

```diff
@@ -0,0 +1,94 @@
+package no.nav.familie.ks.sak.barnehagelister
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import no.nav.familie.ks.sak.barnehagelister.BarnehagelisteVarslingService.Companion.PATH_ENHETSNR_TIL_KOMMUNENR_OVERSIKT
+import no.nav.familie.ks.sak.barnehagelister.domene.BarnehagebarnRepository
+import no.nav.familie.ks.sak.barnehagelister.epost.EpostService
+import no.nav.familie.ks.sak.common.exception.Feil
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.io.File
+import java.time.LocalDateTime
+
+@Service
+class BarnehagelisteVarslingService(
+    val barnehageBarnRepository: BarnehagebarnRepository,
+    val epostService: EpostService,
+) {
+    @Scheduled(cron = "0 0 6 * * ?")
+    @Transactional
+    fun sendVarslingOmNyBarnehagelisteTilEnhet() {
+        val kommunerSendtForFørsteGangSisteDøgn = finnKommunerSendtInnSisteDøgn()
+        val enheterSomSkalVarslesTilKommuner = kommunerSendtForFørsteGangSisteDøgn.groupBy { finnEnhetForKommune(it) }
+
+        if (enheterSomSkalVarslesTilKommuner.isNotEmpty()) {
+            logger.info("Sender epost for nye kommuner i barnehagelister: $kommunerSendtForFørsteGangSisteDøgn")
+
+            enheterSomSkalVarslesTilKommuner
+                .map { (enhetsNr, kommuneNrTilhørendeEnhet) -> finnKontaktEpostForEnhet(enhetsNr) to kommuneNrTilhørendeEnhet.map { finnKommuneNavnForKnr(it) }.toSet() }
+                .forEach { (epostAdresse, kommunerTilhørendeEnhet) ->
+                    epostService.sendEpostVarslingBarnehagelister(epostAdresse, kommunerTilhørendeEnhet)
+                }
+        }
+    }
+
+    // Bruker kommuneNr i stedet for kommunenavn siden det er mindre mulighet for feilskriving
+    @Transactional
+    fun finnKommunerSendtInnSisteDøgn(): Set<String> {
```

Comment:
Denne kan kanskje være `private`? 

```suggestion
    private fun finnKommunerSendtInnSisteDøgn(): Set<String> {
```